### ISSUE TITLE
Broken: (BD)

### ISSUE THREAD

**Issue by Ox3rror** [iptv-org/iptv#13509]
### Broken Link

https://iptv-org.github.io/iptv/countries/bd.m3u

### What happened to the stream?

Not loading

### Possible Replacement (optional)

https://us170.jagobd.com:447/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/channel24-sg-e8e.stream/tracks-v1a1/mono.m3u8

### Notes (optional)

All Bangladesh channel broken. Infinity loading and blank screen. But, In Jagobd (ex : https://www.jagobd.com/channel24) website working as well. 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream)

**Comment by Carlinhos027** [comment]
https://iptv-org.github.io/iptv/countries/bd_jagobd.m3u

II think this is the list you are referring to

**Comment by Ox3rror** [comment]
> https://iptv-org.github.io/iptv/countries/bd_jagobd.m3u
> 
> II think this is the list you are referring to

No, the link you provided shows me error. In bangladesh playlist shows black screen and no information.
